net/natmap/patches/100-remove-commit-id.patch

```diff
@@ -0,0 +1,15 @@
+--- a/build.mk
```

Comment:
But there's a lot of patches in this repo without descriptions. I think it's obvious what this patch do.

Reply:
What do you mean about `test.sh`?

Reply:
I know it might be obvious for you, and for me, but not for others. The patch should have a description. It is pity that other patches do not have it, but they should have it. It's wrong, but we should not continue with the bad approach.

About test.sh, you might want to take a look at this repository, how others are doing that. Basically, it ensures you that your package is run-tested inside the GitHub Actions. The ``test.sh`` prevents that it does not segfault while checking for help, version, etc.

Reply:
I renamed the patch file to `100-dont-get-commit-id-from-git.patch`. What do you think?

Reply:
Added `test.sh`.

Reply:
Naming is good, but still it is missing the formal stuff - author, subject, description, etc.

Reply:
OK, updated.

Reply:
This is a trivial diff, having a proper name is enough. From my side it's not worthy to waste time / bytes for that "normal stuff".

Reply:
Well, the description should contain that it fixes this:

```
make[3]: *** [Makefile:74: build/hev-stun.o] Error 1
<command-line>: error: "COMMIT_ID" redefined [-Werror]
<command-line>: note: this is the location of the previous definition
cc1: all warnings being treated as errors
make[3]: *** [Makefile:74: build/hev-sock.o] Error 1
```

Otherwise, package is not compiled.

Reply:
It's redefined because of line 29.

Reply:
Yes, it is, but that's not IMHO correct. Well, I would say, that this is not good at all. I'm sure that there is a better solution than providing patch, which is ugly hack and because of that, we have errors... :shrug:

Reply:
There's no error. Simple things are getting complex here.